src/consumers.ts

```diff
@@ -167,15 +167,25 @@ class SecurityEngine extends AnalysisConsumer implements DiagnosticProducer
             let diagSeverity;
 
             if (this.vulnerabilityCount == 0 && this.advisoryCount > 0) {
-                diagSeverity = DiagnosticSeverity.Information; 
+                diagSeverity = DiagnosticSeverity.Information;
             } else {
                 diagSeverity = DiagnosticSeverity.Error;
             }
 
+            let msg = `${this.context.name.value}: ${this.context.version.value}`;
+            msg += `\nKnown security vulnerability 🔑: ${this.vulnerabilityCount}`;
+            msg += `\nSecurity advisory 🔨: ${this.advisoryCount}`;
+            // Show exploit information only for Registered users
+            msg += this.exploitCount != null ? `\nExploits 💣: ${this.exploitCount}` : "";
+            msg += `\nHighest severity ⚡: ${this.highestSeverity}`;
+            // Do not show the recommendation version for Free users in case of only security advisories
+            // Show recommendation for Registered Users.
+            msg += (this.changeTo && (this.vulnerabilityCount > 0 || this.exploitCount != null)) ? `\nRecommendation 💉: use version ${this.changeTo}.` : "\nRecommendation 💉: No recommended version.";
```

Comment:
```suggestion
            const recommendedVersion = this.changeTo || "N/A";
            const exploitCount = this.exploitCount || "unavailable";
            const msg = `${this.context.name.value}: ${this.context.version.value};
Known security vulnerability 🔑: ${this.vulnerabilityCount}
Security advisory 🔨: ${this.advisoryCount}
Exploits 💣: ${exploitCount}
Highest severity ⚡: ${this.highestSeverity}
Recommendation 💉: ${recommendedVersion}`;
```

Reply:
PTAL!

Reply:
now that even parag has confirmed that we should not have any of these emojis, can you please remove them and create a fresh clean PR or commit